### DISCUSSION TITLE
Combo or Grouped Product missing feature

### DISCUSSION THREAD

**Discussion by magedfathi** [Blair2004/NexoPOS#1017] [Gastro 4.x Ideas & Issues]
while creating group product consist of two product, the first one is normal product and the second even has multiple unit or has modifiers so the normal process while choosing the combo product from the pos, the system will ask for the unit type or which modifier customer will select but this is not happening and while creating combo product i have to select specific modifier or unit.

let us see this scenario which is there in mac or kfc or any restaurant :
i have combo product consist of:
1- chicken
2- Pepsi (7up or cocacola or pepsi or....) >>> can be achieved on nexo by unit group called pepsi type which has many units pepsi, 7up.
3- Rice (white rice, brown rice, ....) >> can be achieved same the second way or by creating modifier group called rice type and white rice and brown rice products and assign them two rice type modifier group.  

so while cashier punching the combo product he has to choose the pepsi type and rice type and no way no achieve that  because during configuring the combo product we have to select the product unit or modifier type which will be the default on POS screen

**Comment by Blair2004** [comment]
That exists since 4.8.0
https://github.com/Blair2004/NexoPOS-4x/releases/tag/v4.8.0

**Reply by magedfathi** [reply]
Kindly check the following screen shoots which explaining my point

![combo issue1](https://user-images.githubusercontent.com/87772530/186781233-b1d07c32-0dfb-48e4-af91-fb2a3597249b.png)

![combo issue2](https://user-images.githubusercontent.com/87772530/186781343-4eac3838-68c9-4d54-8b3e-8054502a0a46.png)

the first two photos shown that the combo product (group product) is created with one product + product with unit group (Tea) with two units Green Tea and Red Tea " i used that way to achieve variant products) + product has modifiers



![combo issue3](https://user-images.githubusercontent.com/87772530/186781772-c9bc6a14-df24-4cc1-b690-0be83c54c04e.png)

 this photos is shown how it is displaying in kitchen screen with the name of group product without shown the ingredients  ##issue1
and i dont know how it will be shown in the printed paper to kitchen because still i didnt finish all tests before proceeding to this step  but if it is the same without ingredients so this will be issue#2 because kitchen team will not know what they are going to cook

![combo issue4](https://user-images.githubusercontent.com/87772530/186782109-689f3755-cd6d-4bcd-8e87-029cef5f46e2.png)

this screen shown how it is appearing on the POS without ingredients and cashier is not able to select the modifiers or product units

**Reply by Blair2004** [reply]
Hi, 
This is part of the material module under development. The grouped feature can't be used for that purpose.
We'll keep you posted on the progress.
